email_recovery/lib/email_recovery/email_recovery_status.dart.dart

```diff
@@ -0,0 +1,18 @@
+import 'package:json_annotation/json_annotation.dart';
+
+enum EmailRecoveryStatus {
+  @JsonValue('waiting')
+  waiting,
+
+  @JsonValue('inProgress')
+  inProgress,
+
+  @JsonValue('completed')
+  completed,
+
+  @JsonValue('failed')
+  failed,
+
+  @JsonValue('canceled')
+  canceled,
```

Comment:
IMO, there is no need for `JsonValue` here because the enum value already matches the key json object